composer.json

```
@@ -4,6 +4,7 @@
     "homepage": "http://toba.siu.edu.ar",
     "require": {
         "php": ">=8.1.0",
+        "ext-mbstring": "^8.1",
```

```suggestion
        "ext-mbstring": "*",
```

Suele ser sin especificar la versión de la extensión. O encontraste algún detalle ?